src/main/kotlin/no/nav/syfo/dialogmotestatusendring/domain/DialogmoteStatusendring.kt

```diff
@@ -26,6 +27,7 @@ data class DialogmoteStatusendring constructor(
 
     companion object {
         fun create(kDialogmoteStatusEndring: KDialogmoteStatusEndring) = DialogmoteStatusendring(
+            uuid = UUID.randomUUID(),
```

Comment:
Må bruke 
`ZoneId.of("Europe/Oslo").rules.getOffset(this)`
i stedet for ZoneOffset.UTC siden det er det isdialogmote gjør når den lager disse hendelsene.